### PR TITLE
You missed one

### PR DESCRIPTION
**Pull request by aidenwallis**
Noticed a few were missing, should catch them all (except localization now!)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,6 +133,6 @@ manual deploy.
 
 ## ❤ OBS Developers
 
-At its core, Streamlabs OBS is powered by the [OBS](https://obsproject.com/)
+At its core, Streamlabs Desktop is powered by the [OBS](https://obsproject.com/)
 engine. We want to thank all of the developers over at the OBS project for
 their years of tireless hard work, without which Streamlabs OBS wouldn't exist today.
```
Comment (leon-richardt) [inline README.md:138]:
```suggestion
their years of tireless hard work, without which Streamlabs Desktop wouldn't exist today.
```

**Review by leon-richardt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aidenwallis) [reply README.md]: fixed

**Comment by andycreeth** [comment]
Thank you - actually just made the same edits in the README before seeing this PR, so going to close this.  Please note that this name change is a work in progress and it will happen gradually in stages - we are working on it.

**Comment by aidenwallis** [comment]
Got it! Just wanted to raise, thanks for responding!

**Pull request closed without merging**